### DISCUSSION TITLE
Ajv validation for the custom-plugin-lib

### DISCUSSION THREAD

**Discussion by ianFar96** [mia-platform/community#39] [General]
Hi,

I'm currently working on a service and I noticed the validation doesn't go as I would expect. Then I found I have the exact same problem as reported here https://github.com/mia-platform/custom-plugin-lib/issues/231, are there any news on that issue? Did somebody already solved it in another way?

Thanks.

**Comment by fredmaggiowski** [comment]
Hi @ianFar96 I think there has been no update on the issue. Let's move the conversation there if you need help on the matter or want to propose a pull request to fix the bug :)

**Comment by peppemanzi** [comment]
I see that the linked issued has been closed. @ianFar96 I will close this discussion too.
